### ISSUE TITLE
Unable to create instances with  RabbitMQ - Direct routing topology (quorum queues) transport using Powershell Commands

### ISSUE THREAD

**Issue by soujay** [Particular/ServiceControl#3728]
#### Description
As per the [document](https://docs.particular.net/servicecontrol/installation-powershell#servicecontrol-instance-cmdlets-and-aliases-adding-an-instance-version-4), instances can be created using the  `'New-ServiceControlInstance'` cmdlet. But while trying to create an instance that uses RabbitMQ - Direct routing topology (quorum queues) transport , the following error is thrown and instance is not created.
`'New-ServiceControlInstance : Sequence contains no matching element'`

#### Expected behavior
It should be possible to create new instances that uses RabbitMQ - Direct routing topology (quorum queues) via the PowerShell commands 

#### Actual behavior
Unable to create new instances that uses RabbitMQ - Direct routing topology (quorum queues) via the PowerShell commands and it throws a "Sequence contains no matching element" error.

### Steps to reproduce

1. Open ServiceControl Management Powershell in admin mode
2. Run the following command
```
$serviceControlInstance = New-ServiceControlInstance `
   -Name 'Test.DEV.ServiceControl' `
   -InstallPath C:\ServiceControl\Bin `
   -DBPath C:\ServiceControl\DB `
   -LogPath C:\ServiceControl\Logs `
   -Port 44334 `
   -DatabaseMaintenancePort 44335 `
   -Transport 'RabbitMQ - Direct routing topology (quorum queues)' `
   -ConnectionString 'host=localhost;username=guest;password=guest' `
   -ErrorQueue errormq `
   -ErrorRetentionPeriod 10:00:00:00
```
3. The following error is thrown
```
New-ServiceControlInstance : Sequence contains no matching element
At line:1 char:27
+ $serviceControlInstance = New-ServiceControlInstance `
+                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [New-ServiceControlInstance], InvalidOperationException
    + FullyQualifiedErrorId : System.InvalidOperationException,ServiceControl.Management.PowerShell.NewServiceControlI
   nstance
```

**Comment by ramonsmits** [comment]
@soujay does this mean the workaround it to use SCMU or is that also affected?

**Comment by soujay** [comment]
Released [ServiceControl 4.32.3](https://github.com/Particular/ServiceControl/releases/tag/4.32.3)
